CHANGELOG.md

```diff
@@ -19,6 +19,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 <!-- ## Unreleased -->
 
+## [0.17.0] - 2021-08-09
```

Comment:
Keep it under "Unreleased". There's a commented header above that you can uncomment.

Reply:
i think i did it this time around